### ISSUE TITLE
Update API Docs: Notification Endpoint

### ISSUE THREAD

**Issue by mjones-oddball** [department-of-veterans-affairs/notification-api#860]
/v2/notifications/{notification_id}

**Comment by trevor2718** [comment]
Folks, 

```
{
  "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
  "body": "string",
  "completed_at": "2022-11-23T20:21:50.480Z",
  "created_at": "2022-11-23T20:21:50.480Z",
  "created_by_name": "string",
  "email_address": "user@example.com",
  "line_1": "string",
  "line_2": "string",
  "line_3": "string",
  "line_4": "string",
  "line_5": "string",
  "line_6": "string",
  "phone_number": "+16502532222",
  "postage": "string",
  "postcode": "string",
  "recipient_identifiers": [
    {
      "id_type": "VAPROFILEID",
      "id_value": "string"
    }
  ],
  "reference": "string",
  "scheduled_for": "2022-11-23T20:21:50.480Z",
  "sent_at": "2022-11-23T20:21:50.480Z",
  "sent_by": "string",
  "status": "created",
  "subject": "string",
  "template": {
    "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
    "uri": "string",
    "version": 0
  },
  "type": "sms"
}
```
This is what is on the API example, you get a `notification` by calling this API with a `notification id`, I probably need to split this request to 2 by type (sms and email). but you can associate both email and sms to a notification or not?

**Comment by trevor2718** [comment]
I had a response on Slack, thank you. Now waiting on instructions for testing push notifications from @jakehova

**Comment by mjones-oddball** [comment]
@trevor2718 isn't the push notifications a separate task that is not committed to this sprint? Are the changes for this ticket awaiting code review from the team? If so you can take another ticket from to-do.

**Comment by trevor2718** [comment]
@mjones-oddball Of course there are changes for SMS and email already. For the Push, I believe it's a part of the requirement. Just FYI - ![image.png](https://images.zenhubusercontent.com/63272882382270b4b8b445b0/b33031c2-177b-4452-ad70-99ab9e4bff18)
If you don't mind, I can open WIP PR.

**Comment by trevor2718** [comment]
@mjones-oddball No worries. I am so sorry for confusing you. Will do!

**Comment by mjones-oddball** [comment]
@trevor2718 No sorry needed, I did that all by myself :). I just spoke with @k-macmillan and he believes that push notifications don't actually create a notification object like email/sms. Which I did not know. You can probably quickly validate and then push up your other changes for review!

**Comment by trevor2718** [comment]
I am not able to test push functionality at this time. However, push is currently not related to notification object so moving forward with review.

**Comment by cris-oddball** [comment]
approved
